package.json

```diff
@@ -79,7 +79,7 @@
     "ts-jest": "29.1.2",
     "ts-loader": "9.5.1",
     "typescript": "5.3.3",
-    "vanilla-framework": "4.9.0",
+    "vanilla-framework": "4.14.0",
```

Comment:
Sorry, I've just noticed one more thing, I think the new Vanilla release has caused the top nav to wrap onto two lines (the Sign In/user's name is on a new line in the demo). Can you take a look at the breakpoint for different screen sizes?